cypress/support/pages/laptopPage.cy.js

```diff
@@ -0,0 +1,31 @@
+class ClickOnTheLaptops {
+  selectLaptopsCategory() {
+    cy.contains('Laptops').click();
+  }
+
+  selectSonyLaptop() {
+    cy.get('[class="hrefch"]')
+      .contains('Sony vaio i7')
+      .click();
+  }
+
+  clickOnTheAddBtn() {
+    cy.get('[class="btn btn-success btn-lg"]')
+      .contains('Add to cart').click();
+    cy.on('window:alert', (str) => {
```

Comment:
better to remove this cy.on from here and create new method to assert error message

Reply:
this assert in the purchaseForm.cy.js file